### DISCUSSION TITLE
How to set IP for additional network interface in different VLAN

### DISCUSSION THREAD

**Discussion by DanielK37** [Security-Onion-Solutions/securityonion#9917] [General]
Hi all,

I searched discussions already and found many posts trying to add a nic to montoring. But I don't want to add it to monitoring but add a different NIC separately from the management interface in a different VLAN where I can send syslog/sysmon and the like to. I am running on VMWare installed from ISO, latest version yesterday. I have added the NIC in vSphere and I can see it in the SO VM but don't know where to set the manual IP for it. I cannot use DHCP for it because it will set an additional gateway, making the management interface unavailable. So the question is basically how I can manually configure the IP address for an additional network interface. Thanks in advance.

Daniel

**Comment by cm-ops** [comment]
You can use `nmcli` or `nmtui` to set manually set the IP address, etc.
